mdcli/publish.go

```diff
@@ -48,7 +48,10 @@ func (body *PublishErrorBody) UnmarshalJSON(b []byte) error {
 // Publish is a command line interface for publishing a local delivery config
 // to be managed by Spinnaker.
 func Publish(opts *CommandOptions, force bool) (int, error) {
-	fmt.Printf("It is recommended to update your delivery config by committing it to your repository and not by calling the publish command. For more details: http://go.netflix.com/publish-managed-delivery-config")
+	fmt.Fprintln(opts.Stdout, "It is recommended to update your delivery config by committing it to your")
+	fmt.Fprintln(opts.Stdout, "repository and not by calling the publish command. For more details:")
+	fmt.Fprintln(opts.Stdout, "	http://go.netflix.com/publish-managed-delivery-config")
+
```

Comment:
Just to double check ... we will print this message when initializing new projects b/c publish is called behind the scenes.  Not sure if printing this message will confuse users during the `init` workflows.